### ISSUE TITLE
addon does not pronounce the Turkish ı character

### ISSUE THREAD

**Issue by enessaribas** [pumper42nickel/eloquence_threshold#7]
STR:
1. Type the turkish ı character, iether on its own or as part of a word, such as ısı.
Actual: the character is read as a question mark
expected:  the character should be read as i, which is the case in Jaws and previous versions of eloquence. Attempting to fix this by creating a rule in the voice dic causes the synthesizer to pronunce words with capatilized I such as GUI  as g ui.

**Comment by burakyuksek** [comment]
Hi,
To fix that incorrect pronunciation, you need to make entries for both ı and İ.

**Comment by enessaribas** [comment]
Actually no. You can fix this by creating an entry for ı and unchecking case sensetive. If you do not, it will read words with capatilized I differently.
